pkg/cnb/cnb_metadata.go

```diff
@@ -18,11 +19,21 @@ type BuildpackMetadata struct {
 }
 
 type BuilderImageMetadata struct {
-	Buildpacks []BuildpackMetadata `json:"buildpacks"`
+	Buildpacks []BuildpackMetadata    `json:"buildpacks"`
+	Stack      lcyclemd.StackMetadata `json:"stack"`
+}
+
+type Stack struct {
```

Comment:
What are we going to use this for?

Reply:
This is so that we can get the actual run image that is configured inside the builders `stack.toml` and get it's complete reference

Reply:
This is to extract the RunImage that is specified inside the `stack.toml` of the builder and then try and resolve it to a SHA